ci/constants.py

```diff
@@ -5,12 +5,13 @@ class TargetPython(Enum):
     python2 = 0
     python3crystax = 1
```

Comment:
Maybe we should remove `python3crystax`?

Reply:
Yes feel free :+1: